python/paddle/tensor/linalg.py

```diff
@@ -5134,3 +5136,64 @@ def ormqr(x, tau, y, left=True, transpose=False, name=None):
     result = matmul(Q, y) if left else matmul(y, Q)
 
     return result
+
+
+def cholesky_inverse(x, upper=False):
```

Comment:
```suggestion
def cholesky_inverse(x: Tensor, upper: bool = False):
```

这个 API 现在是否可以直接开启类型提示试试呢？标题上可以添加 `[Typing]`，现在 CI 已经添加了类型提示 approval 检查，如果这个 API 前置标注工作尚不满足，可以暂时直接 approve